tls/s2n_psk.c

```diff
@@ -99,6 +99,25 @@ S2N_RESULT s2n_psk_clone(struct s2n_psk *new_psk, struct s2n_psk *original_psk)
     return S2N_RESULT_OK;
 }
 
+S2N_RESULT s2n_resumption_psks_remove(struct s2n_connection *conn)
+{
```

Comment:
I know we wrote a method almost exactly like this but for external PSKs while iterating on s2n_connection_append_psk. We didn't end up keeping it, but maybe it'd be worth preemptively make the type you're searching for an argument, since that'd be a super easy change?